tasks/go_test.py

```diff
@@ -757,6 +758,18 @@ def get_impacted_packages(ctx, build_tags=None):
         for file in files
         if file.endswith(".go") or file.endswith(".mod") or file.endswith(".sum")
     }
+
+    # If non-go modified files are included in a folder contained in a package we consider the
```

Comment:
Seems like this comment is not finished

Reply:
Indeed it was not, I just updated it in https://github.com/DataDog/datadog-agent/pull/22955/commits/43536b66208378573953774be89bca26ff5276f0